source/NVDAObjects/IAccessible/MSHTML.py

```diff
@@ -954,15 +954,15 @@ def setFocus(self):
 
 	def _get_table(self):
 		if self.role not in (controlTypes.ROLE_TABLECELL,controlTypes.ROLE_TABLEROW) or not self.HTMLNode:
-			raise NotImplementedError
+			return None
 		HTMLNode=self.HTMLNode
 		while HTMLNode:
 			nodeName=HTMLNode.nodeName
 			if nodeName:
 				nodeName=nodeName.upper()
 			if nodeName=="TABLE": return MSHTML(HTMLNode=HTMLNode)
 			HTMLNode=HTMLNode.parentNode
-		raise NotImplementedError
+		return None

```

Comment:
I wonder whether this should be logged

Reply:
IMHO, it shouldn't: This `None` is not an error, but rather a piece of information.
If the object is not part of a table, the other IAccessible implementations in NVDA simply return `None` without a notice.